internal/db/schema/migrations/oss/postgres/79/01_auth_oidc_prompt.up.sql

```diff
@@ -0,0 +1,105 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+
+-- auth_oidc_prompt_enm entries define the supported oidc auth method prompts.
+create table auth_oidc_prompt_enm (
+  name text primary key
+    constraint only_predefined_auth_oidc_prompts_allowed
+    check (
+      name in (
+        'none',
+        'login',
+        'consent',
+        'select_account'
+      )
+    )
+);
+
+insert into auth_oidc_prompt_enm (name) values
+  ('none'),
+  ('login'),
+  ('consent'),
+  ('select_account');
+
+-- auth_oidc_prompt entries are the prompts allowed for an oidc auth method.
+create table auth_oidc_prompt (
```

Comment:
should there be some sort of immutable trigger for this table that defines
what's mutable if anything?